### PR TITLE
Rework loading of models into workspace

### PR DESCRIPTION
**Pull request by MoritzWeber0**
- Add support for entrypoints: Users can provide the path to the `aird` file via environment variable.
- Load models via the EASE CLI instead of the onStartup method
- Generate project description dynamically (if missing)
- Add support for multiple repositories (and the `GIT_REPOS_JSON` environment variable)
- Clone models in Python instead of bash
- Add tests for read-only container

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -74,40 +80,69 @@ all: \
 base:
 	docker build -t $(DOCKER_PREFIX)$@ base
 
-capella/base:
+capella/base: base
```
Comment (amolenaar) [inline Makefile:83]: Good to have the parent image encoded in the Makefile.

**Review by amolenaar — COMMENTED** [review]
Nice improvement and simplification. The read-only tests do not work for me unfortunately.

Should we set up a build job to ensure the containers build?

File: readonly/load_models.py [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        )
+
+
+def generate_project_file(project: dict[str, str]) -> None:
+    # More information:
+    # https://help.eclipse.org/latest/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Freference%2Fmisc%2Fproject_description_file.html&cp%3D2_1_3_11
+
+    project_description_file: pathlib.Path = project["location"] / ".project"
+    if not project_description_file.exists():
+        name = next(project["location"].glob("*.aird")).stem
```
[the remark this answers is not in the document]
Comment (MoritzWeber0) [reply readonly/load_models.py:102]: How should we handle this case then? I think a warning would be great, but we have to have to show it to the user then.

**Comment by MoritzWeber0** [comment]
> Nice improvement and simplification. The read-only tests do not work for me unfortunately.

Could you try it again with the latest version (after adding https://github.com/DSD-DBS/capella-dockerimages/pull/52/commits/cb7f5478af20ce4267759212e4ff3b9fb74e1ae4)? It works fine on my machine. I assume you're running it on outdated base images.

> Should we set up a build job to ensure the containers build?

If you mean `pytest`, it would be a bit overkill I'd say. I think we can add it to the GH actions. I'll do it on Monday.

File: readonly/load_models.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (amolenaar) [reply readonly/load_models.py:102]: I was also looking at this, but did not come up with a useful solution. I suppose the capella context requires an aird file of sorts.

Let's leave it for now, and see how it goes.

**Pull request merged**